pkg/webhooks/utils.go

```diff
@@ -63,3 +63,12 @@ func getApplicableKindsForPolicy(p *v1alpha1.Policy) []string {
 	}
 	return kinds
 }
+
+func contains(ruleNames []string, ruleName string) bool {
```

Comment:
we have another function "findKind" in pkg/engine/utils.go, that does the same. Check if a string is contained in a [] string.
we can create a pkg/utils folder to contain these helper functions and use them. It has been used in the #202 branch.